app/javascript/components/shared_components/search/SearchBar.jsx

```diff
@@ -1,20 +1,39 @@
-import React from 'react';
+import React, { useEffect, useMemo } from 'react';
 import PropTypes from 'prop-types';
+import debounce from 'lodash.debounce';
```

Comment:
Is there a good reason of including a library for a functionality that we can implement?

Reply:
Would take a lot of refactoring, as commented above

Reply:
I wasn't referring to the code that we have but rather the use of library for one functionality that we can implement, 
But after giving a second look, my bad you were using only including one module of the library and according to our dependencies we're already using lodash (which is expected for such a lib),
So, overall that makes sense we can come back to this later because the lodash debounce is somehow causing us to introduce complexity,